src/backend/routers/chat.py

```diff
@@ -116,7 +121,9 @@ async def chat(
 
     user_id = request.headers.get("User-Id", None)
     agent_id = chat_request.agent_id
-
+    # warning: process_chat currently mutates chat_request.model
+    req_model = chat_request.model[:]
```

Comment:
same here

Reply:
@scott-cohere I see you added [this](https://github.com/cohere-ai/cohere-toolkit/blob/df07e0c73d1b82a55e380285c5813efcce6cecad/src/backend/services/chat.py#L117) a few weeks ago. Do we still need it? Or can we use the model specified by the user?